ndc-models/src/expression.rs

```diff
@@ -112,14 +123,14 @@ pub enum ComparisonValue {
         /// and each subsequent index refers to the collection outside
         /// its predecessor's immediately enclosing `Expression::Exists`
         /// expression.
+        /// Only used if the 'query.exists.named_scopes' capability is supported.
         scope: Option<usize>,
     },
-    Scalar {
-        value: serde_json::Value,
-    },
-    Variable {
-        name: VariableName,
-    },
+    /// A scalar value to compare against
+    Scalar { value: serde_json::Value },
+    /// A value to compare against that is to be drawn from the query's variables.
+    /// Only used if the 'query.variables' capability is supported.
```

Comment:
As all the types become increasingly complex, this is going to be really helpful for new connector authors to know what they can initially stub out or mark with `todo!` etc.

Reply:
My thoughts exactly. 😄 I'm already losing track and I know the spec well.